mmv1/templates/terraform/pre_create/vertex_ai_models.go.tmpl

```diff
@@ -0,0 +1,9 @@
+var postRequestType string
+
+if _, ok := d.GetOk("source_model"); ok {
+	postRequestType = ":copy"
+
+	// These are not valid fields for the copy request on create
+	delete(obj, "labels")
```

Comment:
If labels isn't in the schema (at least for this PR) let's remove any reference to it in the code

```suggestion
```